django/river/settings/prod.py

```diff
@@ -0,0 +1,3 @@
+from .base import *
+
+ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS") and os.environ.get("ALLOWED_HOSTS").split(",") or []
```

Comment:
Is `os.environ.get("ALLOWED_HOSTS", [])` equivalent?

Reply:
Not equivalent ;) 
What you suggest expects : `ALLOWED_HOSTS=["api:8000", "localhost"]`.
What is currently expected : `ALLOWED_HOSTS=api:8000,localhost`